index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="https://kit.fontawesome.com/75d86a86f1.js" crossorigin="anonymous"></script>
+        <script src="script.js" defer></script>
+        <link rel="stylesheet" href="styles.css">
+        <title>Youth Empowering Africa Summit </title>
+        </head>
+    <body>
+        <header class="header">
+          <section class="top">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a href="#">Log out</a>
+            <a href="#">My page</a>
+            </section>
+            <nav class="navigation-section">
+                <i class="fa-sharp fa-solid fa-bars"></i>
+                <a href="index.html"><img src="images/YCA Main Logo.png" class ="home-logo" alt="logo"></a>   
+                  <ul class="nav-items">
+                      <i class="fa-solid fa-x"></i>
+                      <li><a href="about.html" class="about-us-link nav-link">About</a></li>
+                      <li><a href="" class="disabled nav-link">Program</a></li>
+                      <li><a href="" class="disabled nav-link">Join</a></li>
+                      <li><a href="" class="disabled nav-link">Sponsors</a></li>
+                      <li><a href="" class="disabled nav-link">News</a></li>
+                      </ul>                
+            </nav>
+            <section class="header-contents">
+              <div>
+                <p id="welcome-text">"Hello! sharing world"</p>
+                <h1 class="header-title">YOUTH EMPOWERING AFRICA SUMMIT</h1></div> 
+                <p class="header-description">A joyful celebration believing in value of openness and sharing, creating a positive change width
+                    people from over 55 countries in AFRICA is taking place in Rwanda.</p>
+                   
+                <div class="place-and-time">   
+                <p class="time-frame"> 2023.7.3(MON) ~ 7(FRI)</p> 
+                <p class="place">@Kigali Arena, Kigali convention center and more</p></div>
+                </section>   
+        </header>
+        <main>
+         <section class="main-program">   
+           <div class="main-program-content"> 
+            <h2>Main Program</h2>
+              <hr/>
+           </div>
+           <section class="main-program-card">
+              <div class="card">
+                <img src="images/lecture.png">
```

Comment:
Kindly add alt attribute so that your app would be accessible by other devices. Kindly apply this to all the img tags that don't have the attribute.

Reply:
Fixed.